prisma/schema.prisma

```diff
@@ -8,11 +8,10 @@ generator client {
 }
 
 model User {
-  username  String @id
-  firstName String
-  lastName  String
-  email     String @unique
-
+  username    String       @id
+  displayName String
+  email       String       @unique
+  group      String
```

Comment:
Please from `prisma format` to clean up this formatting.

Reply:
Thanks. Done!